<?xml version="1.0" encoding="UTF-8"?>
<ui version="4.0">
 <class>GpLatLon</class>
 <widget class="QDialog" name="GpLatLon">
  <property name="geometry">
   <rect>
    <x>0</x>
    <y>0</y>
    <width>668</width>
    <height>722</height>
   </rect>
  </property>
  <property name="windowTitle">
   <string>Output Graph</string>
  </property>
  <property name="sizeGripEnabled">
   <bool>true</bool>
  </property>
  <property name="modal">
   <bool>true</bool>
  </property>
  <layout class="QGridLayout" name="gridLayout">
   <item row="0" column="0">
    <layout class="QHBoxLayout" name="hlTrkName">
     <item>
      <widget class="QLabel" name="lblTrack">
       <property name="minimumSize">
        <size>
         <width>70</width>
         <height>20</height>
        </size>
       </property>
       <property name="text">
        <string>Track Name:</string>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dspTrkName">
       <property name="minimumSize">
        <size>
         <width>200</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsName0">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
     <item>
      <widget class="QLabel" name="lblDate">
       <property name="minimumSize">
        <size>
         <width>40</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string>Date:</string>
       </property>
       <property name="alignment">
        <set>Qt::AlignRight|Qt::AlignTrailing|Qt::AlignVCenter</set>
       </property>
      </widget>
     </item>
     <item>
      <widget class="QLabel" name="dspTrkDate">
       <property name="minimumSize">
        <size>
         <width>120</width>
         <height>0</height>
        </size>
       </property>
       <property name="text">
        <string/>
       </property>
      </widget>
     </item>
    </layout>
   </item>
   <item row="2" column="0">
    <layout class="QHBoxLayout" name="hlButton">
     <item>
      <widget class="QPushButton" name="pbClose">
       <property name="minimumSize">
        <size>
         <width>80</width>
         <height>28</height>
        </size>
       </property>
       <property name="text">
        <string>&amp;Close</string>
       </property>
       <property name="shortcut">
        <string>Alt+C</string>
       </property>
       <property name="default">
        <bool>true</bool>
       </property>
      </widget>
     </item>
     <item>
      <spacer name="hsButton0">
       <property name="orientation">
        <enum>Qt::Horizontal</enum>
       </property>
       <property name="sizeHint" stdset="0">
        <size>
         <width>40</width>
         <height>20</height>
        </size>
       </property>
      </spacer>
     </item>
    </layout>
   </item>
   <item row="1" column="0">
    <widget class="GpMapPlot" name="gllPlot" native="true">
     <property name="sizePolicy">
      <sizepolicy hsizetype="Fixed" vsizetype="Fixed">
       <horstretch>0</horstretch>
       <verstretch>0</verstretch>
      </sizepolicy>
     </property>
     <property name="minimumSize">
      <size>
       <width>640</width>
       <height>640</height>
      </size>
     </property>
    </widget>
   </item>
  </layout>
 </widget>
 <customwidgets>
  <customwidget>
   <class>GpMapPlot</class>
   <extends>QWidget</extends>
   <header location="global">hdr/gpmapplot.h</header>
   <container>1</container>
  </customwidget>
 </customwidgets>
 <resources/>
 <connections>
  <connection>
   <sender>pbClose</sender>
   <signal>clicked()</signal>
   <receiver>GpLatLon</receiver>
   <slot>doClose()</slot>
   <hints>
    <hint type="sourcelabel">
     <x>89</x>
     <y>551</y>
    </hint>
    <hint type="destinationlabel">
     <x>202</x>
     <y>445</y>
    </hint>
   </hints>
  </connection>
 </connections>
 <slots>
  <slot>doClose()</slot>
 </slots>
</ui>
